contracts/Moloch.sol

```diff
@@ -151,6 +151,18 @@ contract Moloch is Ownable {
         proposalQueue.finishProposal(members, guildBank, votingShares, lootToken, GRACE_PERIOD_SECONDS);
     }
 
+    function donateWei() public payable {
+        address(guildBank).transfer(msg.value);
+    }
+
+    function getWei() public returns(uint256) {
+        return address(guildBank).balance;
+    }
+
+    function donateTokens(ERC20 tokenAddress, uint amount) public {
```

Comment:
Should we have a "getTokens" function also?

Reply:
Yes, I was thinking about - will work on that next.